Documentation~/index.md

```diff
@@ -0,0 +1,157 @@
+# Package documentation guides
```

Comment:
@jasons-unity can/should we delete this?

Reply:
This has also already been addressed